pkg/checks/external/main.go

```diff
@@ -1026,6 +1028,15 @@ func (ext *Checker) createPod() (*apiv1.Pod, error) {
 	// enforce various labels and annotations on all checker pods created
 	ext.addKuberhealthyLabels(p)
 
+	// Get ownerRefernece for the kuberhealthy pod
+	ownerRef, err := util.GetOwnerRef(ext.KubeClient, ext.Namespace)
+	if err != nil {
+		ext.log("Unable to get the ownerRef from pod", err)
+	} else {
+		// Sets OwnerRefernces on all checker pods
+		p.OwnerReferences = ownerRef
+	}
```

Comment:
Hmm after @jonnydawg and I look through this a bit, I believe we want to return the error instead of just logging it

Reply:
My thought was that it isn't worth not creating the pod just because you can't find the ownerRef so that's why I did a log instead of a return.

But I have updated the code to return the error and removed the else.